web/src/models/FormatVideosTime.js

```diff
@@ -1,4 +1,4 @@
-export function formatTime(seconds) {
+export function correctlyFormatTime(seconds) {
```

Comment:
I dont think that we need to have correctly here. the function name should be somehting like formatTimeForVideo